Marlin/Marlin_main.cpp

```diff
@@ -3767,6 +3776,54 @@ inline void gcode_M206() {
 
 #endif // FWRETRACT
 
+/**
+ * M211: Enable or disable software endstops
+ */
+#if ENABLE_M211 == true
+inline void gcode_M211() {
```

Comment:
You can indent the whole function between the `#ifdef` and `#endif`. The complete coding standards are written up (finally) at https://github.com/MarlinFirmware/Marlin/wiki/Coding-Standards